File: module-1/13_Inheritance_Abstract_Classes/student-exercise/java/src/main/java/com/techelevator/SquareWall.java
package com.techelevator;

public class SquareWall extends RectangleWall {

	public SquareWall(String name, String color, int sideLength) {
		super(name, color, sideLength, sideLength);
	}
	
	public String toString() {
		return getName() +  " (" + getLength() + "x" + getHeight() + ") square";
	}
}

File: module-1/13_Inheritance_Abstract_Classes/exercise-final/dotnet/AbstractExercise/SquareWall.cs
namespace AbstractExercise
{
    public class SquareWall : RectangleWall
    {
        public SquareWall(string name, string color, int sideLength) : base(name, color, sideLength, sideLength) { }

        public override string ToString()
        {
            return $"{Name} ({Length}x{Height}) square";
        }
    }
}